tests/test_plot_api.py

```diff
@@ -5525,6 +5525,42 @@ def test_chart_and_indicators():
     )
 
 
+def test_tornado():
+    def gen_rand_df(product_names, neg=False):
+        """
+        Generate random data for testing
+        """
+        df_size = 10
+        rng = np.random.default_rng()
+
+        values = rng.uniform(low=10, high=30, size=df_size)
+        if neg:
+            values = -1 * values
+
+        return pd.DataFrame({
+            'value': values,
+            'name': product_names,
+        })
+
+    product_names = ["Cheeseburger", "Fried Chicken", "Pasta Carbonara", "Caesar Salad", "Fish and Chips",
+                 "Beef Stroganoff", "Pizza Margherita", "Grilled Salmon", "Taco Salad", "Crispy Calamari",
+                 "Shrimp Scampi", "Garlic Bread", "Sizzling Fajitas", "Chicken Teriyaki", "French Onion Soup",
+                 "Miso Ramen", "Crab Cakes", "Lobster Bisque", "Crispy Tofu Bowl", "Philly Cheesesteak",]
+    # Random data
+    df_pos = gen_rand_df(product_names[:10])
+    df_neg = gen_rand_df(product_names[10:20], neg=True)
+
+    s.plt.tornado(
+        df_neg=df_neg, df_pos=df_pos,
+        neg_dataops_mod={
+            'itemStyle': {
+                'color': 'blue',
+            },
+        },
+        order=0, menupath='Tornado'
```

Comment:
It would be better to define the menu_path at the beginning of the function and following the pattern of the other charts, also the name of the function can be printed for clarity when executing, see the other functions for example